### ISSUE TITLE
Feature request: delete-backup command to accept a list of backups

### ISSUE THREAD

**Issue by spintower** [thelastpickle/cassandra-medusa#410]
There are currently 3 ways to delete multiple backups: purge by max age, purge by max count, or run `delete-backup` multiple times. We have full and differential backups that have different retention: full backups are kept for 1 year and differentials for 30 days. This makes purging by age and purging by max count (which is essentially purge by age with age determined by number of existing backups) not useful for us. Running `delete-backup` is very time-consuming because of the orphan deletion process. The purge command goes through orphan deletion only once even when deleting multiple backups, but `delete-backup` has no option to specify multiple backups at the same time.

Could you change `delete-backup` command to accept multiple backups?

```
medusa delete-backup --backup-name backup1 backup2 backup3
```

**Comment by adejanovski** [comment]
Hi @spintower,

this sounds like a nice improvement indeed. 
If you feel like pushing a PR for this, I'd be happy to review it.

**Comment by paulojmdias** [comment]
Hi @spintower @adejanovski,

Also, support do `medusa purge --all-nodes` isn't a good option? If you have some node replaced with another hostname or even if you replace all nodes by new ones (rolling update for example) the old backups will never be purged. It will only be deleted if we have a script to list the backups with `--all-nodes` option and calculate which backups should be deleted based on age or date.

**Comment by adejanovski** [comment]
@paulojmdias, totally. I've created another ticket to track this work: #447 

The PR fixing the initial issue was merged a while ago. Closing the ticket.
